doc/source/getting_started/develop_pymapdl.rst

```diff
@@ -0,0 +1,337 @@
+===================================
+
+Install the latest version of PyMAPDL in development mode:
+
+
+.. code:: console
+
+    cd pymapdl
+    pip install pip -U
+    pip install -e .
+
+
+If you are going to do testing, you need to install the testing dependencies:
+
+
+.. code:: console
+
+    pip install -e '.[tests]'
+
+
+Develop PyMAPDL
+===============
+
+.. epigraph:: *Now it is time to develop PyMAPDL!*
+
+Developing code in a repository, particularly when using version control systems
+like Git, involves a set of essential guidelines to ensure efficient
+collaboration, code management, and tracking changes. Here are the main
+guidelines for developing code in a repository:
+
+#. **Use Branches**: Create branches for different features, bug fixes, or
+   experiments. This keeps changes isolated and facilitates parallel
+   development.
+
+#. **Write Descriptive Commit Messages**: Provide clear and concise commit
+   messages that explain the purpose and context of the changes. Follow a
+   consistent style.
+
+#. **Commit Frequently**: Make small, meaningful commits frequently. Avoid
+   making a large number of unrelated changes in a single commit.
+
+#. **Pull Before You Push**: Always update your local branch with the latest
+   changes from the remote repository before pushing your own changes to avoid
+   conflicts.
+
+#. **Use Pull Requests (PRs)**: Use PRs to submit your changes for review.
+   This allows for discussion and validation before merging into the main branch.
+
+#. **Write good documentation**: Maintain clear and up-to-date documentation for your
+   contribution or changes, including README files, comments in code, and relevant project
+   documentation. Make sure you update the related documentation as well.
+   Follow `numpydoc <numpydoc_>`_ convention for documenting code.
+
+#. **Test Your Changes**: Thoroughly test your contributions to ensure they work
+   as expected. If applicable, create or update unit tests which runs on the 
+   continuous integration/continuous deployment (CI/CD) pipelines to catch issues early
+   and ensure reliable deployments. See `Unit testing`_ for more details.
+
+#. **Respect Code Style and Standards**: Follow consistent code style
+   guidelines and adhere to coding standards specific to your language or
+   framework.
+
+#. **Collaborate and Communicate**: Communicate with team members, provide
+   updates on your progress, and resolve any conflicts promptly.
+
+#. **Ask for help**: To ensure code quality, identify issues, and share knowledge,
+   PyMAPDL developers are always happy to assist you through code reviews.
+   In case you need help, or guidance, mention ``@ansys/pymapdl-maintainers`` in a comment
+   so they can be notified.
+
+By following these guidelines, you can ensure smooth and organized code
+development within a repository, fostering collaboration, code quality, and help the
+project maintainers.
+
+
+.. _ref_unit_testing_contributing:
+
+Unit testing
```

Comment:
Again, this section seems like something that should be added to the _PyAnsys developer's guide_ and referenced in the documentation for every library.

Reply:
I agree. I will try to port it there and reference here in later PRs. :)